Application.cpp

```diff
@@ -0,0 +1,72 @@
+#include "json.hpp"
+#include <iostream>
+#include <fstream>
+#include <string>
+#include <iomanip>
+using namespace std;
+using json = nlohmann::json;
+
+class Application 
+{
+public:
+	string GetUserInput()
```

Comment:
Create Application.h file and declare Application class and its functions there.
Include it in Application.cpp and define it.